packages/extension-store-locator/README.md

```diff
@@ -11,6 +11,10 @@ A [PWA Kit](https://github.com/SalesforceCommerceCloud/pwa-kit) extension that a
 
 ```sh
 npm install @salesforce/extension-store-locator
+
+# Also: 
+# - install the peer dependencies listed in the package.json
+# - see the Peer Dependancies section below for any other steps (e.g. make sure your app uses CommerceApiProvider component)
```

Comment:
Maybe we can talk about this in our weekly meeting, but what is the current experience for base project developers installing extensions.. does npm install peers or not and is it annoying to have to do that if it doesn't

Reply:
Aside: we also talked about how it'd be nicer if the store-locator can automatically add the CommerceApiProvider if needed, similar to how it's done for the Chakra UI.